app/controllers/orders_controller.rb

```diff
@@ -10,6 +10,22 @@ def create
     @order = Order.new(user: current_user, total: current_user.cart.total_cart_price)
     if @order.save
       @order.purchase_line_items
+
+      customer = Stripe::Customer.create(
+        email:  @order.user.email,
+        source: params[:stripeToken]
+      )
+      @order.card_token = customer.id
+
+      amount = @order.total
```

Comment:
If I'm not mistaken, you shouldn't need to continue calling `order` as an instance variable (`@order`). After you define it on line 10, you can just call `order` afterwards. I still don't understand why but I've had it drilled in my head that
![screen shot 2016-02-04 at 5 39 13 pm](https://cloud.githubusercontent.com/assets/9851764/12832150/43669502-cb66-11e5-8e21-44a9c62a2dc1.png)